### PR TITLE
Added Language Support Page to 2.0 Docs

### PR DESCRIPTION
**Pull request by b-quachtran**
**Proposed changes**:
- Added Language Support Page to 2.0 Docs

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Review by erohmensing — CHANGES_REQUESTED** [review]
I don't know if this is the right spot for this. I think this page is mostly about making sure that people who are starting know that Rasa can support their language. Here's what I'd propose: 

Move this to the getting started section (which we removed bc it had only one entry, but can add back in: 

Building Assisnants
  Getting Started
    Installation
    Language Support

Make this page itself less technical, along the lines of "you can build assistants in any lanuguage you want. Rasa provides components that can learn directly from your data, ... 
You can also incorporate existing language models...." And then we can link to the suggested pipelines for the different recommendations (not sure if those will continue to live in "tuning your nlu model"

I'm interested to see if @karen-white has ideas for this page, as it sort of seems like something that fits in the marketing FAQ as well, but we do need search in the docs to turn something up along these lines

**Comment by erohmensing** [comment]
Considering this is a highly requested page, I'll add it as is, but would like us to reconsider the content and where it goes as part of the "finish documentation updates" for Q4

File: docs/docs/language-support.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+:::note
+Even more so when training word embeddings from scratch, more training data will lead to a
+better model! If you find your model is having trouble discerning your inputs, try training
+with more example sentences.
+:::
+
+
+## Pre-trained Word Vectors
```
Comment (erohmensing) [inline docs/docs/language-support.mdx:43]:
```suggestion
## Using Pre-trained Language Models
```

File: docs/docs/language-support.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+If you can find them in your language, pre-trained word vectors are a great way to get started with less data,
+as the word vectors are trained on large amounts of data such as Wikipedia.
```
Comment (erohmensing) [inline docs/docs/language-support.mdx:46]:
```suggestion
If you can find them in your language, language models with pre-trained word vectors are a great way to get started with less data,
as the word vectors are trained on large amounts of data such as Wikipedia.
```

**Review by erohmensing — APPROVED** [review]

**Comment by karen-white** [comment]
@erohmensing we include a multi-language mention in our FAQ page (need to update the link though, it looks like): https://info.rasa.com/faqs#multi-lang

agree it would be good to think about how we can enrich this page and make it discoverable :)

**Pull request merged**